docs/concepts/how-celestia-works/data-availability-layer.md

```diff
@@ -71,7 +71,7 @@ consequence of using a 2-dimensional Reed-Solomon encoding scheme. Alternatively
 DAS could be designed with a standard (i.e., 1-dimensional) Reed-Solomon encoding,
 where the original data is split into k  chunks and extended with k additional
 chunks of parity data. Since the block data commitment is the Merkle root of the
-2k resulting data chunks, light nodes no longer need to download O(n) bytes to
+2k resulting data chunks, light nodes no longer need to download 4k bytes to
```

Comment:
[for reviewers] This was previously incorrect because light nodes need to download `O(sqrt(n))` bytes to validate block headers. But I think it's clearer here to use `4k` given the problem statement at the beginning of this paragraph:

> The requirement of downloading the 4k intermediate Merkle roots is a
consequence of using a 2-dimensional Reed-Solomon encoding scheme.